packages/elements/src/config/elements/config.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  BasicElement,
+  TemplateResult,
+  html
+} from '@refinitiv-ui/core';
+import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
+import { state } from '@refinitiv-ui/core/decorators/state.js';
+import { property } from '@refinitiv-ui/core/decorators/property.js';
+import { VERSION } from '../../version.js';
+import { efConfig, type Config as ConfigType, type ConfigIcon } from '../helpers/context.js';
+import { provide } from '@lit-labs/context';
+
+@customElement('ef-config', { theme: false })
+export class Config extends BasicElement {
+
+  /**
+   * Element version number
+   * @returns version number
+   */
+  static get version (): string {
+    return VERSION;
+  }
+
+  @provide({ context: efConfig })
+  @state()
+    config: ConfigType = {};
```

Comment:
We don't provide a fully structure default config here. Does that create any problems with consumers?